src/quo2/components/community/divider_label.cljs

```diff
@@ -0,0 +1,62 @@
+(ns quo2.components.community.divider-label
+  (:require [quo.react-native :as rn]
+            [quo2.components.markdown.text :as text]
+            [status-im.ui.components.icons.icons :as icons]
+            [quo2.foundations.colors :as colors]))
+
+(def chevron-icon-container-width 20)
+
+(def chevron-icon-container-height 20)
+
+(defn divider-label
+  "label -> string
+   chevron-position? -> :left, :right
+   counter-value? -> number
+   tight -> boolean
+   blur -> boolean"
+  [{:keys [label chevron-position? counter-value? tight blur] :or {label "Welcome"
```

Comment:
better to do not have default label, or show something like `Warning:Empty Label`